### DISCUSSION TITLE
[Feature Request] Label-naming list dropdown when naming segments

### DISCUSSION THREAD

**Discussion by cesarave** [mifi/lossless-cut#2074] [Ideas]
I would like to propose a new feature for LosslessCut - a label dropdown menu when naming new segments. While working with LosslessCut, I noticed that adding this feature would significantly enhance the user experience. It will help to quickly and easily apply predefined labels to their video segments. Instead of manually typing out common segment names, I could select from a customizable list of frequently used labels, saving time and reducing errors. This would be especially helpful for those of us who regularly work with similar types of segments, such as chapters, scenes, or specific event markers. Implementing this feature would streamline the editing process, making LosslessCut even more efficient :). Is this something that can be done?

**Comment by mifi** [comment]
Hi. added your suggestion here: #427

**Reply by cesarave** [reply]
I appreciate very much the quick response! Looking forward to it :)
